tests/unit/s2n_self_talk_session_id_test.c

```diff
@@ -311,12 +348,25 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_set_cache_retrieve_callback(config, cache_retrieve_callback, session_cache));
         EXPECT_SUCCESS(s2n_config_set_cache_delete_callback(config, cache_delete_callback, session_cache));
 
+        /* Although we disable session ticket, as long as session cache
+         * callbacks are binded, session ticket key storage would be initialized
+         */
+        GUARD(s2n_config_set_session_cache_onoff(config, 1));
+        GUARD(config->wall_clock(config->sys_clock_ctx, &now));
+        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(config, ticket_key_name, strlen((char *)ticket_key_name), ticket_key, strlen((char *)ticket_key), now/ONE_SEC_IN_NANOS));
```

Comment:
you will overrun on `strlen(ticket_key)`;  you want `sizeof(ticket_key)`

Reply:
Thanks for pointing this out!